assets/base.css

```diff
@@ -1039,11 +1039,20 @@ summary::-webkit-details-marker {
   width: calc(50% - var(--grid-mobile-horizontal-spacing) / 2);
```

Comment:
@wiktoriaswiecicka @katycobb @melissaperreault 

For the sections that have `enable swipe on mobile`, I wonder if we should either

1) Not have the mobile setting
2) Have a note that the setting won't work when the enable swipe is activated
3) Have the columns changing numbers even when the swiping is working

Here is a quick video of what I mean: https://screenshot.click/11-58-nlcn8-bs2u1.mp4

Reply:
👋 @bredowmax, thanks for sharing your thoughts. Yes, for mobile we are only allowing 1 or 2 columns for the sections in the PR description :) 

This comment was related to the when we have swipe on mobile turned on.

Reply:
@sofiamatulis In that scenario the number of columns, it should inform how many items are in view, then the peek.

i.e.
- 1 column =  We see one card and a peek
- 2 columns = we see 2 cards and a peek

Reply:
I could see one column slider with peekaboo effect being nice on mobile 🤔 🤷  I wouldn't be against it. If your products are about imagery, then it'd be nice to be able to show a bigger product card.